core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11971,4 +11971,50 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     assertThat(toSql(root, DatabaseProduct.ORACLE.getDialect()), isLinux(expectedOracle));
   }
 
+
+  @Test public void testForGetBitFunction() {
+    final RelBuilder builder = relBuilder();
+    final RexNode getBit = builder.call(SqlLibraryOperators.GETBIT,
+        builder.literal(8), builder.literal(3));
+    final RelNode root = builder
+        .scan("EMP")
```

Comment:
can we not use .values() instead of .scan over here as we don't require a table name in the output query? Check in all test cases and remove wherever it is not required

Reply:
Done